### DISCUSSION TITLE
How to disable tooltips in Menu of collapsed Sider?

### DISCUSSION THREAD

**Discussion by sklinov** [ant-design/ant-design#34616] [Q&A]
I'm building a custom menu using Sider and Menu/Submenu components.
Basically, I need to replace regular behaviour of Menu.Item contents with `<a ...>` type of links and add custom Tooltips.
Right now, if Sider is in `collapsed` mode - the built-in tooltip is shown and I don't see a way to disable it.

**Comment by 23I-K** [comment]
Hi, add a prop {title=''} to Menu.Item, for disabling tooltip.

https://ant.design/components/tooltip

" The Tooltip can be disabled by setting title='' "
